pycqed/analysis/plotting_tools.py

```diff
@@ -1,3 +0,0 @@
-from pycqed.analysis.tools.plotting import *
-import logging
-logging.warning('plotting_tools is deprecated, use analysis.tools.plotting')
```

Comment:
considering this warning, do we know of any function being used from this file? otherwise we could just delete the whole file

Reply:
This file was indeed deleted in the pull request :)